pulumi/github/repos/pipelines/testpipeline.py

```diff
@@ -0,0 +1,212 @@
+# NOTE => are tests from PHP
+# TODO Convert => to actual tests https://www.pulumi.com/docs/using-pulumi/testing/
+# https://github.com/pulumi/examples/blob/74db62a03d013c2854d2cf933c074ea0a3bbf69d/testing-unit-py/test_ec2.py
+import pulumi
+import pulumi_github as github
+
+NAME = "testpipeline"
+
+TOPICS = [
+    "nextflow",
+    "pipelines",
+    "nf-test",
+    "modules",
+    "nf-core",
+    "dsl2",
+    "workflows",
+]
+
+# Names of required CI checks. These are added to whatever already exists.
+# public $required_status_check_contexts = [
+#     'pre-commit',
+#     'nf-core',
+REQUIRED_CI_CHECKS = [
+    github.RepositoryRulesetRulesRequiredStatusChecksRequiredCheckArgs(
+        context="pre-commit",
+        integration_id=0,
+    ),
+    github.RepositoryRulesetRulesRequiredStatusChecksRequiredCheckArgs(
+        context="nf-core",
+        integration_id=0,
+    ),
+]
+
+CORE_TEAM_ID = 2649377
+MAINTAINERS_TEAM_ID = 4462882
+
```

Comment:
```suggestion
repo = github.Repository.get(pipeline, f"nf-core/{pipeline}")
```
let's first query the old repo, to not overwrite certain fields